src/v/wasm/tests/wasm_transform_test.cc

```diff
@@ -53,6 +53,6 @@ TEST_F(WasmTestFixture, WorksWithCpuProfiler) {
     std::vector<ss::cpu_profiler_trace> traces;
     ss::engine().profiler_results(traces);
     for (const auto& t : traces) {
-        std::cout << t.user_backtrace << "\n";
+        fmt::print("{}\n", t.user_backtrace);
```

Comment:
Eh is this even supposed to be checked in?

Reply:
By me? Or the original stuff with `cout`? Yeah it crossed my mind that it's weird. Who am I to judge though?